app/assets/js/rtMedia.backbone.js

```diff
@@ -795,6 +809,11 @@ jQuery( function( $ ) {
 					return message;
 				};
 			} );
+				
+			// This function will store hidden values of Media ID in Array
+        	function store_array_in_hidden_field(){
+				jQuery("#rtm_bpm_uploaded_media").attr("value", msg_media_files.toString());
```

Comment:
`jQuery("#rtm_bpm_uploaded_media").attr("value", msg_media_files.toString());` could be 

`jQuery( '#rtm_bpm_uploaded_media' ).attr( 'value', msg_media_files.toString() );`